gramine/README.md

```diff
@@ -0,0 +1,75 @@
+## How to run demo with Gramine
+
+1. Start a SPS container:
+```shell
+docker run -d -v /opt/multiple/sessions:/graphene/workplace/sessions -v /opt/secret-prov/certs/:/graphene/workplace/certs -p 8080:8080 -p 4433:4433 -e SPS_USERNAME=admin -e SPS_PASSWORD=admin --name iexec-sps iexechub/iexec-sps:bf3bb00-dev
+```
+
+
+2. Build your app:
+```shell
+docker build -t gramine-tee-worker-post-compute:latest -f gramine/Dockerfile  .
+```
+Please note the `measurement` value.
+
+
+3. To add a session to the SPS, run the following after having filled both env var:
+```shell
+SESSION_ID=<define your custom session id>
+MEASUREMENT=<set previous retrieved measurement>
+SPS_IP=<set SPS IP>
+
+TASK_ID=<set task ID>
+RESULT_STORAGE_CALLBACK=<set whether you use callback>
+RESULT_SIGN_WORKER_ADDRESS=<set worker address>
+RESULT_SIGN_TEE_CHALLENGE_PRIVATE_KEY=<set TEE challenge private key>
+RESULT_STORAGE_TOKEN=<set result storage token>
+RESULT_STORAGE_PROXY=<set result storage proxy>
+
+curl --location --request POST 'localhost:8080/api/session/' \
+--header 'Authorization: Basic YWRtaW46YWRtaW4=' \
```

Comment:
Maybe add a note to tell this must be updated depending on how the SPS has been configured at startup

Reply:
I've replaced `localhost:8080` with a var. See https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/54/commits/443a67615c9d309fa7e2771dc0b85dcedab6092b :)

Reply:
I think JB talks about `auth = base64(user:login)`